src/pokemon-service/utils/file.go

```diff
@@ -0,0 +1,64 @@
+package utils
+
+import (
+	"bufio"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+type file struct {
+	filename string
+}
+
+type File interface {
+	ReadAllFileLines() ([]string, error)
+	AppendLineToFile(string) error
+}
+
+// Gets a new instance of the file utils
+func NewFileUtil(fn string) File {
+	return &file{filename: fn}
+}
+
+// Read all the lines in a file
+func (f file) ReadAllFileLines() ([]string, error) {
+	file, err := os.Open(f.filename)
+	if err != nil {
+		log.Fatal("Failed to open", err)
```

Comment:
Avoid using fatal here, you can just return the error, and the layer which calls this layer, stop the application

Reply:
I intended to use log.Panic just to log the error right there and return the error, made that change